src/ontology/mondo-edit.obo

```diff
@@ -331247,6 +331249,7 @@ subset: orphanet_rare {source="Orphanet:100002"}
 subset: rare
 synonym: "soft tissue perineurioma" EXACT [Orphanet:100002]
 xref: GARD:19730 {source="Orphanet:100002"}
+xref: NCIT:C6912 {source="https://orcid.org/0009-0001-6494-4831"}
```

Comment:
Source `MONDO:equivalentTo` is missing

Reply:
Can you add these - I overlooked them (I added these by hand from the spreadsheet we worked on). Sorry; Thanks :D

Reply:
I will add these, no problem! :-)